src/Auth0.ManagementApi/Clients/ClientGrantsClient.cs

```diff
@@ -1,23 +1,27 @@
-using Auth0.Core.Http;
 using Auth0.ManagementApi.Models;
 using Auth0.ManagementApi.Paging;
+using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace Auth0.ManagementApi.Clients
 {
     /// <summary>
-    /// Contains all methods to call the /client-grants endpoints.
+    /// Contains methods to acess the /client-grants endpoints.
```

Comment:
```suggestion
    /// Contains methods to access the /client-grants endpoints.
```

Reply:
**Nit (throughout):** I would not include the endpoint path in the docblocks, just the name that's used in the docs. Maybe a link to that docs page/section as well?

Reply:
Will do that in the code-generated version. Don't want to spend any more cycles on code that will be replaced.